templates/feed-stylesheet.xsl

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
-    <xsl:output method="html" version="1.0" encoding="UTF-8" indent="yes"/>
+<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd">
+    <xsl:output method="html" version="1.0" encoding="ISO-8859-1" indent="yes"/>
```

Comment:
Can we please stick to UTF-8, which is a common standard in multi-language web development.